climada_petals/entity/exposures/crop_production.py

```diff
@@ -363,9 +362,9 @@ def from_isimip_netcdf(cls, input_dir=None, filename=None, hist_mean=None,
             value_tmp = np.squeeze(area_crop[irr_val]*hist_mean_dict[irr_val][idx_mean])
             value_tmp = np.nan_to_num(value_tmp) # replace NaN by 0.0
             exp.gdf['value'] += value_tmp
-        exp.tag = Tag(description="Crop production exposure from ISIMIP"
-                                  f" {CROP_NAME[crop]['print']} {irr}"
-                                  f" {yearrange[0]} {yearrange[-1]}")
+        exp.description=("Crop production exposure from ISIMIP"
+                        f" {CROP_NAME[crop]['print']} {irr}"
+                        f" {yearrange[0]} {yearrange[-1]}")
```

Comment:
Similar topic: I find it bad practise to set an attribute that is not defined in the constructor. (I know that `Exposures` combines several bad practices when it comes to constructors but we don't have to make it worse)

The `description` attribute is also never documented anywhere.